website/docs/components/accordion/accordion-a11y.md

```diff
@@ -22,22 +22,18 @@ The following list describes roles and attributes that component already has.
-| -------- | ------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `region` |                           | `div`    | Creates a landmark region that contains the currently expanded accordion panel.                                                                                                                                             |
-| `button` |                           | `div`, `button` | The `button` role is for clickable elements that trigger a response when activated by the user. Adding `role="button"` tells the screen reader the element is a button, but provides no button functionality.               |
-|          | `aria-expanded="true"`    | `button` | Set to `true` when the Accordion panel is expanded, otherwise set to `false`.                                                                                                                                               |
-|          | `aria-controls="ID"`      | `button` | Points to the `ID` of the panel which the header controls.                                                                                                                                                                  |
-|          | `aria-labelledby="IDREF"` | `div`    | Defines the accessible name for the region element. References the accordion header button that expands and collapses the region. `region` elements are required to have an accessible name to be identified as a landmark. |
-|          | `aria-disabled`             |          | The `aria-disabled` state indicates that the element is perceivable but disabled, so it isn’t editable or otherwise operable. |
+| Component / element           | Role & attributes         | Usage                                                                                                                                     |
+| ----------------------------- | ------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `Accordion.Item.ToggleButton` | `role="button"`           | Defines element as clickable and can be activated by user, and controls the visibility of accordion's content panel.                      |
+|                               | `aria-expanded="true"`    | Set to `true` when the accordion panel is expanded, otherwise set to `false`.                                                             |
+|                               | `aria-controls="ID"`      | Points to the `ID` of the panel which the toggle button controls.                                                                         |
```

Comment:
```suggestion
|                               | `aria-controls="IDREF"`      | Points to the `ID` of the panel which the toggle button controls.                                                                         |
```
probably should say "IDREF" for consistency with other items?

Reply:
Yeeees! Missed it 🙈 Thank you!